src/styles/global.ts

```diff
@@ -48,18 +48,23 @@ export const breakpoints = {
   sm: '768px' as const,
   md: '1024px' as const,
   lg: '1440px' as const,
+  customBp: '1217px' as const,
 }
```

Comment:
Also I think you could just type the object `as const`, and not need to add it to each row.